scripts/pylib/twister/twisterlib.py

```diff
@@ -1107,7 +1109,7 @@ def handle(self):
 
         logger.debug("Running %s (%s)" % (self.name, self.type_str))
         command = [self.generator_cmd]
-        command += ["-C", self.build_dir, "run"]
+        command += ["-C", self.build_dir, "run_qemu"]
```

Comment:
why this change ?
The `run` target should still default to the first entry in the `SUPPORTED_EMU_PLATFORMS` list.

Which in this case means it should still be possible to just execute `run`.